packages/assets-controllers/src/MultichainAssetsRatesController/MultichainAssetsRatesController.test.ts

```diff
@@ -240,6 +240,24 @@ describe('MultichainAssetsRatesController', () => {
     expect(snapHandler).not.toHaveBeenCalled();
   });
 
+  it('does not update conversion rates if the assets are empty', async () => {
+    const { controller, messenger } = setupController();
+
+    const snapSpy = jest.fn().mockResolvedValue({ conversionRates: {} });
+    messenger.registerActionHandler('SnapController:handleRequest', snapSpy);
+
+    // Publish a selectedAccountChange event.
+    // @ts-expect-error-next-line
+    messenger.publish('MultichainAssetsController:stateChange', {
+      accountsAssets: {
```

Comment:
nit (happy to ignore since this is a test).
Could we satisfy the type by using the `defaultState` from the controller? Or is the types quite complex?

```ts
const mockNewState = { ...defaultState }
mockNewState.accountAssets = { account: [] }
messenger.publish('MultichainAssetsController:stateChange', mockNewState)
```